### ISSUE TITLE
[Bug] UU On-boarding: progressbar må ha et aksessbart navn

### ISSUE THREAD

**Issue by tormoseng** [AtB-AS/webshop#367]
**Beskrivelse av bug**
Progressbar øverst har rollen progressbar. Alle elementer med denne rollen må enten ha satt en ikke-tom aria-label eller en aria-labelledby.

https://dequeuniversity.com/rules/axe/4.1/aria-progressbar-name

**Comment by gorandalum** [comment]
Jeg tenker det er feil at dette elementet har `role="progressbar"`.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role
https://www.digitala11y.com/progressbar-role/

For eksempel står det:
> A progressbar indicates that the user's request has been received and the application is making progress toward completing the requested action

Det står også:
> The progressbar role should be used for an element that displays the progress status for a task that take a long time or consists of several steps.

Men jeg tolker "several steps" som flere steg i en automatisert prosess, f.eks. flere nettverkskall som skjer.

Jeg skriver om denne til å ikke bruke `role="progressbar"`.

**Comment by gorandalum** [comment]
Fikses i lenket PR. Markerer nå med `role="figure"` med en `aria-label` som har steg-informasjonen. Siden det ikke er noe lesbart innhold under figure så leses det på VoiceOver som:
> Steg 2 av 3, samtykker, You are currently on an empty figure

Jeg tror det er greit. Jeg kunne lagt teksten som en ikke-synlig tekst som et underelement av `figure` og brukt `aria-labelledby`, men jeg tror det blir mer knot for brukeren siden det da blir tre elementer å navigere gjennom. Lesingen blir da ca. slik:
> Steg 2 av 3, samtykker, figure start
> Steg 2 av 3 // Selve tekstelementet
> Figure end

**Comment by tormoseng** [comment]
PR testet OK manuelt og med auto a11y-sjekk! Enig med kommentaren over.
